packages/fxa-auth-server/lib/push.js

```diff
@@ -82,34 +89,6 @@ module.exports = function (log, db, config) {
     };
   }
 
-  /**
-   * Reports push errors to logs
-   *
-   * @param {Error} err
-   * @param {String} uid
-   * @param {String} deviceId
-   */
-  function reportPushError(err, uid, deviceId) {
-    log.error(LOG_OP_PUSH_TO_DEVICES, {
-      uid: uid,
-      deviceId: deviceId,
-      err: err,
-    });
-  }
-
-  /**
-   * Reports push increment actions to logs
-   *
-   * @param {String} name
-   */
-  function incrementPushAction(name) {
-    if (name) {
-      log.info(LOG_OP_PUSH_TO_DEVICES, {
-        name: name,
-      });
-    }
-  }
```

Comment:
I've refactored these a little, and also moved them down to the bottom of the file so they're closer to their callers.